src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.ReturnKey.cs

```diff
@@ -2,18 +2,20 @@
 
 using System;
 using System.Threading;
-using Microsoft.CodeAnalysis.Editor.Commands;
+using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Formatting
 {
     internal partial class FormatCommandHandler
     {
-        public CommandState GetCommandState(ReturnKeyCommandArgs args, Func<CommandState> nextHandler)
+        public VSCommanding.CommandState GetCommandState(ReturnKeyCommandArgs args, Func<VSCommanding.CommandState> nextHandler)
         {
             return nextHandler();
         }
 
-        public void ExecuteCommand(ReturnKeyCommandArgs args, Action nextHandler)
+        public void ExecuteCommand(ReturnKeyCommandArgs args, Action nextHandler, CommandExecutionContext context)
         {
             ExecuteReturnOrTypeCommand(args, nextHandler, CancellationToken.None);
```

Comment:
>CancellationToken.None [](start = 58, length = 22)

not cancellable for some reason so I won't change it now (we do plan to catch and log commands not reacting to user cancellation going forward)

Reply:
Filed https://github.com/dotnet/roslyn/issues/24155 to improve it later.